indico/testing/fixtures/session.py

```diff
@@ -18,8 +18,8 @@
 def create_session(db):
     """Return a callable that lets you create a session."""
 
-    def _create_session(event, title):
-        entry = Session(event=event, title=title)
+    def _create_session(event, title, **kwargs):
+        entry = Session(event=event, title=title, **kwargs)
         db.session.add(entry)
         db.session.flush()
         return entry
```

Comment:
Pretty sure someone copied it from a timetable-entry-related fixture back then ;)

```suggestion
        sess = Session(event=event, title=title, **kwargs)
        db.session.add(sess)
        db.session.flush()
        return sess
```